CONTRIBUTING.md

```diff
@@ -840,6 +840,14 @@ reviewer for each component that is significantly affected), that
 change requests have been implemented, that CI is passing, and
 eventually will be the one who merges the PR.
 
+The PR assignee may use their own judgement to decide to merge a PR that
+has not received reviews from all maintainers of affected components,
+depending on how large or controversial the changes to these components
+are. It is also admitted to have an assignee who is not a maintainer of
```

Comment:
"admitted" doesn't sound right imo

Reply:
I think `admissible` is probably what was intended.